modules/virt-removing-virtio-disk-from-vm.adoc

```diff
@@ -15,10 +15,12 @@ Remove the `container-native-virtualization/virtio-win` container disk from the
 .Procedure
 . Edit the configuration file and remove the `disk` and the `volume`.
 +
+[source,terminal]
 ----
 $ oc edit vm <vm-name>
 ----
 +
+.Example output
```

Comment:
I don't think that this needs the `.Example output` header because it's not really output, it's a file that you're editing. The `oc edit` command opens the YAML file in the default text editor, AFAIK. 

@aburdenthehand sanity check?

Reply:
I wasn't sure about this one. I went for the `.Example output` header because the file opens as a result of the `oc edit` command but I can see why it's not really output.

Reply:
Removed the `.Example output` header